### PR TITLE
Rename rootCA to caCert

### PR DESCRIPTION
**Pull request by rene-dekker**
The CRD has been updated.

### CODE REVIEW

**Review by bcreane — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
